GameFiles/IW4x/userraw/scripts/_integration.gsc

```diff
@@ -561,6 +561,18 @@ OnExecuteCommand( event )
         case "Unhide":
             response = self UnhideImpl();
             break;
+        case "LockControls":
+            response = self LockControlsImpl( event.target );
```

Comment:
`self` here is the origin of the command. The action should be executed in the context of the `target`.
This should be `event.target LockControlsImp()`.
Then there is no need to pass the target